src/main/java/com/pusher/client/channel/impl/PresenceChannelImpl.java

```diff
@@ -51,7 +55,51 @@ public User getMe() {
     @Override
     public void onMessage(final String event, final String message) {
```

Comment:
This is the main method that has changed. Instead of calling the parent method, now it uses it's own.